C/c4Base.cc

```diff
@@ -184,7 +184,7 @@ CBL_CORE_API C4LogDomain const kC4WebSocketLog = (C4LogDomain)&websocket::WSLogD
 C4LogDomain c4log_getDomain(const char* name, bool create) noexcept {
     if ( !name ) return kC4DefaultLog;
     auto domain = LogDomain::named(name);
-    if ( !domain && create ) domain = new LogDomain(name);
+    if ( !domain && create ) domain = new LogDomain(name, LogLevel::Info, true);
```

Comment:
I don't know if the default level should be Info or something else like warning.

Reply:
Probably leave it as the default?  Why explicitly change it here I wonder?

Reply:
This is because of 
` explicit LogDomain(const char* name, LogLevel level = LogLevel::Info)`
before this commit, and the new argument, internName, has default false.